app/static/app/js/custom.js

```diff
@@ -1,6 +1,8 @@
-setTimeout(function focusMe() {
+function focusMe() { window.setTimeout( function() {
 	document.getElementById("focus-here").scrollIntoView();
```

Comment:
Throws TypeError: document.getElementById(...) is null.

Use #46 inside **`focusMe`**'s definition and it should be good to go.